docs/02-Install.md

```diff
@@ -1,13 +1,312 @@
+candidates or phantoms.
+
+2. **`Visit_Windows`** table
+
+To populate with visit labels, you can manually insert study-specific 
+information:
+
+
+      INSERT INTO Visit_Windows (Visit_label,  WindowMinDays, WindowMaxDays, OptimumMinDays, OptimumMaxDays, WindowMidpointDays) VALUES ('V1', '0', '100', '40', '60', '50');
+
+If age is a not critical factor in study visit scheduling, define Min value as 
+0, and Max value as 39999.
+
+
+Alternatively, LORIS provides a PHP script in its `tools/` directory 
+`populate_visit_windows.php` that can be used.
+
+
+4. **`mri_scan_type`** and **`mri_protocol`** tables
+
+Ensure your `mri_scan_type` and `mri_protocol` tables contains an entry for 
+each type of scan in the study protocol.
+The `mri_protocol` table is used to identify incoming scans based on their 
+SeriesDescription **OR** scan parameter values (TE, TR, slice thickness, etc). 
+By default, this table is populated with entries for t1, t2, fMRI and DTI, and 
+the columns defining expected scan parameters (e.g. `TE_Range`) are defined very 
+broadly.  
+The `Scan_type` column values are defined in the `mri_scan_type` table 
+(e.g. 44=t1); do not include hyphens, spaces or periods in your 
+`mri_scan_type.Scan_type` column values.
+
+5. **`Config`** table
+
+The `Config` table can also be accessed and customized from the LORIS front-end, 
+under the Admin menu, the `Configuration` module. Here are the configuration
+settings that impact directly or indirectly the pipeline:
+
+Under the section `Study`:
+ * `ImagingUploader Auto Launch`
+ 
+Under the section `Paths`
+ * `Imaging Data`
+ * `LORIS-MRI Code`
+ * `MRI-Upload Directory`
+ * `MINC Files`
+ * `Images`
+ 
+ Under the section `Imaging Modules`
+  * `Patient ID regex`: Used by the DICOM Archive module to show/hide the PatientID info
+  * `Patient name regex`: Used by the DICOM Archive module to show/hide the Patient Name info
+  * `Lego phantom regex`: Used by the DICOM Archive module to show/hide the Patient Name info for phantoms
+  * `Living phantom regex`: Used by the DICOM Archive module to show/hide the Patient Name info for phantoms
+  * `Imaging Browser Tabulated Scan Types`: Used by Imaging Browser to decide what modalities to display the QC status on
+     
+ Under the section `Imaging Pipeline`:
+ * `Loris-MRI Data Directory`: Typically `/data/$PROJECT/bin/mri/`
+ * `Study Name`: Prefix used for all filenames inserted into Imaging Browser and the `files` table
+ * `User to notify when executing the pipeline`: User email address used when notification is to be sent by the pipeline
+ * `Full path to get_dicom_info.pl script`: Typically `/data/$PROJECT/bin/mri/dicom-arhive/get_dicom_info.pl`
+ * `Horizontal pictures creation`
+ * `NIFTII file creation`: Project choice to enable for automated NIFTII file creation
+ * `dcm2mnc binary to use when converting`
+ * `Path to Tarchives`
+ * `Upload creation of candidates`: Creates candidates into LORIS if enabled
```

Comment:
To remain consistent with my rephrasing above if you accept it:
Enable or disable candidate creation into LORIS when running the insertion pipeline

Reply:
ok